demaceo/portfolio
# WebGazer

#### What does  this PR do?
- Creates WebGazer component 
- Adds a side navigation bar within Settings component
- Adds music interests to interestData.js
- Adjusts the website logo
- Adjusts fonts of Bubbles
- Begins working on LightSwitch component
#### How should this be manually tested?
- Click the Settings icon in the top right of the screen
- Click WebGazer 
- Click the photos as they appear
#### Any background context you want to provide?
- A lot of incomplete functionality here.
#### APIs & Additional Resources added in this PR:
- [Google Maps API](https://www.npmjs.com/package/google-maps-react)
- [WebGazer](https://webgazer.cs.brown.edu/#support)
